### DISCUSSION TITLE
tabularray in RTL documents

### DISCUSSION THREAD

**Discussion by Udi-Fogiel** [lvjr/tabularray#360] [Ideas]
Hello, currently there is no support for typesetting tables in RTL context. In [this post](https://tex.stackexchange.com/questions/671877/shifted-hlines-in-tblr-tabular-in-rtl-context) there are few ideas on how to do so with LuaLatex. It would be nice to add that support, and I don't believe  it would take allot of effort to do so, It might need  only a few more conditionals (to test if LuaLatex is runing and testing the text direction) and adding the patchs in the post (there might be a better way to implement this).

**Comment by muzimuzhi** [comment]
related #351
